manifests/config.pp

```diff
@@ -1,22 +1,25 @@
 # == Class: marathon::config
 #
 class marathon::config(
-  $conf_dir_base = '/etc/marathon',
-  $conf_dir_name = 'conf',
-  $owner         = 'root',
-  $group         = 'root',
-  $master        = undef,
-  $zookeeper     = undef,
-  $options       = {},
-  $env_var       = {},
-  $manage_logger = true,
-  $logger        = 'logback',
-  $log_dir       = '/var/log/marathon',
-  $log_filename  = 'marathon.log',
-  $log_level     = 'info',
-  $ulimit        = undef,
-  $java_home     = undef,
-  $java_opts     = '-Xmx512m',
+  $conf_dir_base                    = '/etc/marathon',
+  $conf_dir_name                    = 'conf',
+  $owner                            = 'root',
+  $group                            = 'root',
+  $master                           = undef,
+  $zookeeper                        = undef,
+  $options                          = {},
+  $env_var                          = {},
+  $manage_logger                    = true,
+  $logger                           = 'logback',
+  $log_dir                          = '/var/log/marathon',
+  $log_filename                     = 'marathon.log',
+  $log_level                        = 'info',
+  $ulimit                           = undef,
+  $mesos_auth_principal             = undef,
+  $mesos_auth_secret                = undef,
+  $mesos_auth_secret_file           = '/etc/marathon/.secret',
+  $java_home                        = undef,
+  $java_opts                        = '-Xmx512m',
```

Comment:
Indentation of the `=` a bit messed up here since the parameter names got shorter.